web-testbed/tests/tests_backend/proxies/object_proxies.py

```diff
@@ -0,0 +1,74 @@
+from .base_proxy import BaseProxy
+from .object_proxy import ObjectProxy
+
+
+class AppProxy(BaseProxy):
+    def __init__(self):
+        super().__init__("self.my_objs['__app__']")
+        # super().__init__("self")
+
+
+AppProxy.__name__ = AppProxy.__qualname__ = "App"

```

Comment:
This is a neat trick to get around some dynamic behaviours... it does make me a *little* nervous though. Changing core class metadata like this is something that can have very deep seated consequences down the line.